### DISCUSSION TITLE
libdohj for parsing blocks problem :)

### DISCUSSION THREAD

**Discussion by qlpqlp** [dogecoin/dogecoin#3303] [Help needed]
So Im in contact with some Devs at Ammer Wallet and they have a problem paring blocks using the  libdohj.
They say only started to happen since the recent  network updates.

more precisely it broke the serialization/deserialization of transactions

Anyone know anything about it and how to workaround?

Much thanks in advance :)

**Comment by patricklodder** [comment]
Bugs with libdohj should really go into dogecoin/libdohj. There was no recent network update to block structure tho...

**Reply by qlpqlp** [reply]
I will direct this reply to them :)
